bpf_test.go

```diff
@@ -233,6 +237,26 @@ func checkSocketFilters(t *testing.T, b *elf.Module) {
 	}
 }
 
+func checkPinConfig(t *testing.T, b *elf.Module) {
+	var expectedPaths = []string{
+		"/sys/fs/bpf/gobpf-test/testgroup1",
```

Comment:
same comment as before: parameter `b` is not used there: it could be removed and replaced by `expectedPaths`.